lib/launch-fastify.js

```diff
@@ -28,6 +28,7 @@ const { exportFastifyOptions, exportServiceOptions } = require('./options-extrac
 const statusRoutes = require('./status-routes')
 const metricsRoutes = require('./metrics-routes')
 const swaggerDefinition = require('./swagger-definition')
+const { get } = require('lodash')
```

Comment:
we use to put external imports before local imports (indicatively at line 22)

Reply:
Could we do not import all lodash to only have the get module?

Reply:
That's a fair point, there's no need for lodash, for what matters we could resolve the access with a simple optional chaining